transport/x/grpc/util.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package grpc
+
+import (
+	"fmt"
+	"net/url"
+
+	"go.uber.org/yarpc/internal/procedure"
+)
+
+const defaultMethodName = "__default__"
+
+func procedureNameToServiceNameMethodName(procedureName string) (string, string, error) {
+	serviceName, methodName := procedure.FromName(procedureName)
```

Comment:
so is the idea here to have the grpc method to be:

`/UserService/GetUser`

As opposed to

`/populous/UserService::GetUser`
?

If that is the case, do we have enough granularity to determine which `service` (e.g. Populous) is being called from GRPC's perspective?  And can we route to it?  I think some fringe services might use this service name overload to customize routing (I believe muttley-prober uses it to distinguish muttley-prober-hbe vs muttley-prober)

Not 100% convinced either way, but, whatever we choose we will have to support for the rest of 1.0 (I'd be comfortable landing this into the x branch and refining it)